### DISCUSSION TITLE
Single run localizer

### DISCUSSION THREAD

**Discussion by alireza-kr** [cvnlab/GLMsingle#145] [Q&A] [answered]
Hello

I have a single run localizer and I want to use GLMsingle (or GLMdenoise) to process it. I know that both GLMsingle and GLMdenoise require at least two runs to work. The solution is to artificially split a run into two runs, but I have some questions about the details:

- The length of both artificial runs must be the same, right? In my case, which is a block design with each trial lasting 6 seconds, there might be a trial that spans the middle.
- Is the spilling over of experiment-related effects from the end of the first run to the next a critical issue? How it can affect the results?
- My last question is whether the number of trials for a specific condition should be balanced across artificial runs. For example, is it acceptable if I have two instances of a condition in the first run of a subject and three instances of the same condition in the second run? Or for another subject, if I have one instance in the first run and four instances in the second run?

**Comment by kendrickkay** [comment, marked as answer]
The lengths of the runs do not have to be the same.

The spilling over is suboptimal, but probably not a huge deal.  Basically, you will miss capturing the end of the first run (a lost opportunity).  And, you might be failing to fully model the beginning of the second run...  but probably not a huge deal.  Arguably, there are all sorts of things that are not fully being modeled at the beginning of runs, like the subject getting ready, fidgeting, etc.

The runs don't need to be fully balanced; it should be okay.  I believe this issue is discussed at https://glmsingle.readthedocs.io/en/latest/wiki.html
